client/src/pages/BlogPostPage.tsx

```diff
@@ -32,6 +35,21 @@ export default function BlogPostPage() {
     getEntry();
   }, [entryId]);
 
+  async function handleDelete() {
+    try {
+      const req = {
+        method: 'DELETE',
+      };
+      const res = await fetch(`/api/entries/${entryId}`, req);
```

Comment:
Oh, this is why it works. You should move this code into fetchFunctions.ts (deleteEntry).